packages/react-router/modules/Router.js

```diff
@@ -21,6 +22,11 @@ class Router extends React.Component {
 
   render() {
     const { children } = this.props
+    invariant(
+      children == undefined || React.Children.count(children) === 1,
+      'A <Router> must have at most one child element.'
```

Comment:
Yeah, I would agree that that could be rephrased. I like yours better. Will see what everyone else thinks but happy to reword.